phantom_token.c

```diff
@@ -375,21 +395,45 @@ static ngx_int_t handler(ngx_http_request_t *request)
     introspection_request->headers_in.content_length_n = ngx_buf_size(introspection_request_body_buffer);
 
 #if(NGX_HTTP_HEADERS)
-    module_context->original_accept_header = request->headers_in.accept->value;
+    if (request->headers_in.accept == NULL)
```

Comment:
looks like what's needed here is a static function that takes a request and an accept header value. with that, you could simply call it twice. that would avoid some duplication

Reply:
fixed in 6f379c3dd922d5b57f23e156dd65d73c977a211f.